packages/core/src/families/hedera/validation.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod";
+import { schemaFamilies, schemaTransactionCommon } from "../common";
+
+export const schemaRawHederaTransaction = schemaTransactionCommon.extend({
+  family: z.literal(schemaFamilies.enum.hedera),
+  memo: z.string().optional(),
+});
```

Comment:
The memo field in Live seems to be capped to 100 characters ([code](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/families/hedera/MemoField.jsx#L23)). We should enforce the same rule in Wallet API side as well I think.

```javascript
memo: z.string().max(100).optional(),
```